Recruit-Training/Advanced-Recruit-Training/src/BatteryStateOfChargeService.cpp

```diff
@@ -8,6 +8,9 @@ namespace
 BatteryStateOfChargeService::BatteryStateOfChargeService(double initialStateOfChargePercent)
 : initialStateOfChargePercent_(initialStateOfChargePercent)
 {
+    ampHourUsed=0.0;
+    holdC = QTime(0,0,0,0);
+    holdD = QTime(0,0,0,0);
```

Comment:
What is holdC and holdD, I think it stands for charging vs depleting but can you make it so. For example, isCharging and isDepleting?